runtime/make/Makefile.runtime.include

```diff
@@ -127,7 +127,11 @@ ifeq ($(CHPL_LIB_PIC),pic)
 RUNTIME_DEFS += $(SHARED_LIB_CFLAGS)
 endif
 
-RUNTIME_CFLAGS += $(RUNTIME_DEFS)
-RUNTIME_CXXFLAGS += $(RUNTIME_DEFS) $(CXXFLAGS)
-
+# If compiling for GPU locale add CUDA runtime headers to include path
+ifeq ($(CHPL_LOCALE_MODEL),gpu)
```

Comment:
Think you want `CHPL_MAKE_LOCALE_MODEL` (set [here](https://github.com/chapel-lang/chapel/blob/77e71e6e3d3781ab8fc7f422f5a41cdb432b73cb/make/Makefile.base#L42)) instead of `CHPL_LOCALE_MODEL`. That's the `make` variable that corresponds to the Chapel configuration's `CHPL_LOCALE_MODEL` setting, and will be set even if the latter is not (i.e., if the locale model is specified implicitly, rather than explicitly via the environment).

Reply:
You're correct, updating it to use `CHPL_MAKE_LOCALE_MODEL` makes it so I no longer have to explicitly pass in `-DHAS_GPU_LOCALE` in the `.compopts` file.